### ISSUE TITLE
On my Minisforum HX99G running Bazzite OS, a sunshine stream of Gaming Mode to Moonlight Client results in distorted streamed image. Works fine in Desktop mode

### ISSUE THREAD

**Issue by ZainAjam** [LizardByte/Sunshine#2646]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

- [X] This issue is present in the latest pre-release

### Describe the Bug

When I start a Moonlight connection to my Minisforum Hx99g device running in Gaming Mode with Sunshine installed, I get a weird distorted image streamed. It looks like this on my Moonlight device. This is only a problem in Gaming Mode. Works fine in Desktop mode.

![image](https://github.com/LizardByte/Sunshine/assets/32865278/3b29f68c-c2f0-4440-8a3f-30e43b52af19)

### Expected Behavior

Instead of the stream being in the top left quadrant of the screen, I expect it to fill the screen, and without all the other distorted quadrants of colours around it.

### Host Operating System

Linux

### Operating System Version

Bazzite OS 3.0.1 - 2024-06-07 build

### Architecture

32 bit

### Sunshine commit or version

0.23.1

### Package

Linux - AppImage

### GPU Type

AMD

### GPU Model

AMD Radion 6650M

### GPU Driver/Mesa Version

Unsure

### Config

```Shell
Unsure
```

### Relevant log output

```shell
Unsure
```

**Comment by gschintgen** [comment]
This seems like a client issue, i.e. Moonlight. This bug tracker is for Sunshine.

**Comment by cgutman** [comment]
This is impacted by host-side game resolution, so I don't think it's a Moonlight problem.

**Comment by ShadowMaker** [comment]
This really is a big problem tbh. So you basically have no way to stream any old games having low resolution - they look fine on the Deck host itself, but on the client it keeps alternating between that weird image above and normal view. I can provide a video if this could help, but I think this is quite important for anyone who streams anything which has lesser resolution than Steam Deck native.

**Comment by TariqMK** [comment]
I was able to resolve this on my HX99G running Bazzite.

The issue lies in the fact that that in Gaming Mode, the resolution (in Settings > Display) is set to 'Automatic'. 

In my case I was running my HX99G on a 4K TV, but only using 1080p resolution. When I set the resolution in the above mentioned 'Settings > Display' to 1080p instead of leaving it at 'Automatic', this resolved the issue.

I was then able to connect to the machine in Game Mode from another machine full screen without issue.

In summary, set the resolution in Steam Game Mode settings manually to the resolution you wish to use and try again.

**Comment by ShadowMaker** [comment]
Nope, this was the first thing I tried on my Steam Deck. Mybe HX99G runs Steam differently, idk. But no matter what resolution I set in settings for the game does not change the behavior.

**Comment by xrishox** [comment]
I've only tested this for a few minutes but, I was seemingly able to solve this by enabling developer mode and inside of developer mode checking force composite.

This also solved another major problem I was having where I couldnt see the steam overlay from moonlight clients

( https://github.com/LizardByte/Sunshine/issues/3860#issuecomment-2954267416 thank you @mrpppbtthl5 )
